pkg/controller/operator/common/defaults.go

```diff
@@ -216,15 +216,15 @@ var (
 			{
 				// Auto-upgrade unsupported clusters
 				From:      "1.15.*",
-				To:        "1.16.*",
+				To:        "1.16.13",
 				Automatic: pointer.BoolPtr(true),
 			},
 
 			// ======= 1.16 =======
 			{
 				// Allow to change to any patch version
 				From: "1.16.*",
-				To:   "1.16.*",
```

Comment:
Here you should keep the wildcard, only where we have automatic upgrade we need a fixed version in the `To`